packages/plugins/documentation/controllers/Documentation.js

```diff
@@ -222,11 +222,10 @@ module.exports = {
     strapi.reload.isWatching = false;
     const service = strapi.plugins.documentation.services.documentation;
     const documentationVersions = service.retrieveDocumentationVersions().map(el => el.version);
+
     const {
-      request: {
-        params: { version },
-        admin,
-      },
+      params: { version },
```

Comment:
@petersg83 I have made a small update in the controller as the request.params.version was undefined.